src/run.rs

```diff
@@ -227,6 +232,60 @@ pub(crate) fn run_repl(
         use_color,
     );
 
+    let delta = {
+        let name = "std".to_string();
+        let content = get_standard_library().as_bytes();
+
+        let mut working_set = StateWorkingSet::new(engine_state);
+
+        let start = working_set.next_span_start();
+        working_set.add_file(name.clone(), content);
+        let end = working_set.next_span_start();
+
+        let (_, module, _, _) = parse_module_block(
```

Comment:
Here it would be good to preserve the module comments and pass them to `add_module()` so you can have a nice help message when calling `help std`.

Reply:
should be good in 61436bce4, `help std` gives the comment now